types/jsreport-core/index.d.ts

```diff
@@ -15,20 +18,36 @@ declare namespace JsReport {
     type Recipe = "html";
 
     interface Template {
-        content: string;
+        content?: string;
         engine: Engine | string;
-        helpers: Helpers;
+        helpers?: Helpers;
         recipe: Recipe | string;
+        pathToEngine?: string;
```

Comment:
and here.

Reply:
I don't believe it should be required to pass this path to the engine when defining a Template since I don't believe it's a required field and it's normally supplied via the engine.

https://github.com/jsreport/jsreport-core/blob/b3f907cc3b8a4fe18dc91f48430b2df6e691264c/lib/render/render.js#L54